test/functionalTest/cases/3045_wrong_inclusion_location_metadata_notification/wrong_inclusion_location_metadata.test

```diff
@@ -0,0 +1,275 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Wrong inclusion of "location" metadata in geo:json notifications
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create E with type T 
+# 02. Create subscription for E
+# 03. Trigger notification
+# 04. See notification and make sure the metadata is correct
+#
+
```

Comment:
I would suggest to use more precise name and steps (including echo messages) in this test. For instance

```
--NAME--
Check "location" metadata is not wrongly included in NGSIv2 notifications

--SHELL--

# 01. Create E with type T 
# 02. Create NGSIv2 subscription for E
# 03. Trigger notification
# 04. See NGSIv2 notification and make sure "location" metadata is not included
```

Reply:
Fixed in c76622d